libs/utils/decorate.js

```diff
@@ -19,13 +19,28 @@ let videoLabels = {
 };
 let videoCounter = 0;
 
+const shouldBlockFreeTrialLinks = (a, prefix, parent) => {
+  if (prefix !== '/kr' || (!a.dataset?.modalPath?.includes('/kr/cc-shared/fragments/trial-modals')
+    && !['free-trial', 'free trial', '무료 체험판', '무료 체험하기',
+      '{{free-trial}}', '{{start-free-trial}}', '{{try-for-free}}']
```

Comment:
Wouldn't `free-trial` account for `{{free-trial}}` and `{{start-free-trial}}` as well?